_pages/index.md

```diff
@@ -13,20 +13,56 @@ layout: home
-- Let us know when you are ready to go live and our team will help you [promote the application to production]({% link _pages/production.md %}). We will check you against our production checklist to ensure your application is good to go to production from an administrative and technical standpoint.
-- Be sure to read the [FAQ]({{ site.baseurl }}/support/#frequently-asked-questions) for answers to the most common questions.
+<ol class="usa-process-list usa-prose margin-bottom-4">
+  <li class="usa-process-list__item">
+    <p>
+      Learn about the process the user will go through to access your application and what steps are involved in configuring your app on the <a href="{% link _pages/overview.md %}" class="usa-link">Integration</a> page.
+    </p>
+  </li>
 
+  <li class="usa-process-list__item">
+    <p>
+      Determine your application needs, like the level of proofing, the <a href="{% link _pages/design-guidelines.md %}" class="usa-link">user experience</a> decisions your agency will make, and the <a href="{% link _pages/attributes.md %}" class="usa-link">user attributes</a> that will be requested.
+    </p>
+  </li>
+
+  <li class="usa-process-list__item">
+    <p>
+      Select between <a href="{% link _pages/oidc.md %}" class="usa-link">Open ID Connect</a> (OIDC) or <a href="{% link _pages/saml.md %}" class="usa-link">SAML</a> implementation protocols.
+    </p>
+  </li>
+
+  <li class="usa-process-list__item">
+    <p>
+      Configure your app in the <a href="https://dashboard.int.identitysandbox.gov/" class="usa-link">sandbox dashboard</a> and start <a href="{% link _pages/testing.md %}" class="usa-link">testing</a>! We also have <a href="https://dashboard.int.identitysandbox.gov/" class="usa-link">example apps</a> to get you up and running quickly.
+    </p>
+  </li>
+
+
+  <li class="usa-process-list__item">
+    <p>
+      When you are ready to go live our team will help you <a href="{% link _pages/production.md %}" class="usa-link">promote the application to production</a>. We will check against our production checklist to ensure your application is ready for production from an administrative and technical standpoint.
+    </p>
+  </li>
+</ol>
+
+<hr class="text-primary-light border-solid measure-5 margin-x-0">
+
+<h2 class='margin-top-4'>Integration support for developers</h2>
+<section class="usa-prose">
+  <p>
+    If you are with a government agency partner, check our <a href="{% link _pages/support.md %}" class="usa-link">FAQ</a> page for answers to the most common questions. If you need further technical assistance with an integration, submit a ticket to the <a href="https://zendesk.login.gov/" class="usa-link">Partner Support Help Desk</a> 
+  </p>
+  <p>
+    For help signing in or verifying your identity with Login.gov, please visit the <a href="https://login.gov/help/" class="usa-link">Login.gov Help Center</a> or <a href="https://login.gov/contact/" class="usa-link">contact us</a>.
+  </p>
+</section>
```

Comment:
Minor: Currently these are nested `<section>`, but if the headings are both `h2`, I wonder if they should be sibling sections instead, as well as moving the heading inside the section.

```suggestion
</section>
<section class="usa-prose">
  <h2 class='margin-top-4'>Integration support for developers</h2>
  <p>
    If you are with a government agency partner, check our <a href="{% link _pages/support.md %}" class="usa-link">FAQ</a> page for answers to the most common questions. If you need further technical assistance with an integration, submit a ticket to the <a href="https://zendesk.login.gov/" class="usa-link">Partner Support Help Desk</a> 
  </p>
  <p>
    For help signing in or verifying your identity with Login.gov, please visit the <a href="https://login.gov/help/" class="usa-link">Login.gov Help Center</a> or <a href="https://login.gov/contact/" class="usa-link">contact us</a>.
  </p>
```